### ISSUE TITLE
Unable to view packs

### ISSUE THREAD

**Issue by viveksh1** [omaha-consulting/winstall#154]
Whenever I try to view my own packs, I always get the following error screen. This has been happening for a few days now.
![image](https://github.com/MehediH/winstall/assets/57074714/3b237361-8600-441f-8119-7368bbaa94eb)

**Comment by MysteryDash** [comment]
Yeah unfortunately the website has been borked for a while and I don't know if we'll see a fix.
However, as a workaround, if you go to your Packs and then click on the edition button while watching for your network traffic in your browser, you can catch the https://api.winstall.app/packs/********** call that contains your entire pack data.
Not great, but if you need to retrieve your original pack content, then it's the way to go.

**Comment by vepman** [comment]
same issue here.

**Comment by YannickFuereder** [comment]
same issue here.

**Comment by IP0** [comment]
I'm very curious as to why, this service is using Twitter/X in the first place?
What is the idea behind it, validating humans? I'm just trying to understand.
Also, I really like the improvement compared to winget.run buuuut, with the
currently missing functionality, stability and forced login, I'm switching back.

**Comment by spudpiggy** [comment]
Bump

**Comment by mherrmann** [comment]
I'll be happy to merge a PR that fixes the issue

**Comment by Kmfernan5** [comment]
is it ready?

**Comment by PotentCricket** [comment]
considering this issue is from June and is still happening, I'm assuming we're not going to get a fix and this is essentially a dead website now? - At least for viewing/creating packs?

**Comment by mherrmann** [comment]
I'm the new website owner. Can you help me understand why Packs are so important to you? Can't you just copy the installation commands for the apps you want to a text file?

**Comment by PotentCricket** [comment]
> I'm the new website owner. Can you help me understand why Packs are so important to you? Can't you just copy the installation commands for the apps you want to a text file?

Hey, well apart from convenience it also gives a way for people to share their packs or to find packs, perhaps for gaming or dev work or student apps, whatever it may be. It can be kind of tedious creating a new text file and copy/pasting every single command and also manually adding the &&, sure it's doable but it was nice to be able to just jump onto the website and select a bunch of app and you're done, script/bat/txt whatever your commands are there ready 👌

**Comment by KutaG59** [comment]
Still getting this issue, but with "Recieved error: [object Object]". Assuming this website is just dead and being dragged through mud? new owner doesn't seem to care. Anyone have any other websites that include a packs feature? rather not make 7 different text files, each with 50+ apps (all varying) by copy and pasting every, single, thing.

**Comment by a-fried-pOtaTO** [comment]
Same issue as @KutaG59 
All packs are inaccessible. The error message is below:


Sorry! We could not load this pack.
Recieved error: [object Object]

**Comment by cry8wolf9** [comment]
Will this be fixed at any point soon? I am still seeing the issue when viewing packs

**Comment by PotentCricket** [comment]
> Will this be fixed at any point soon? I am still seeing the issue when viewing packs

At this point unlikely I'd say. The new owner seems uninterested in fixing it and more interested in the "premium" version of this for corporate customers but the looks of things. I'd just move on and find something else to use like scoop perhaps.

**Comment by psydex** [comment]
Bump

**Comment by niu541412** [comment]
Use the edit button, we can still see the package list currently. With the browser developer tool, I've backuped my packageID list.
![image](https://github.com/omaha-consulting/winstall/assets/3370897/8c0d9e5a-fd49-4285-9d09-876a899adb12)

**Comment by KutaG59** [comment]
7 months later......still the same issue.......still the same webpage.

> I'm the new website owner. Can you help me understand why Packs are so important to you? Can't you just copy the installation commands for the apps you want to a text file?

You obviously don't care about the feature, remove it please. It doesn't work, stop hosting it as a feature when its not even viable to use. I don't get what you achieve by allowing the /packs webpage to be accessed if you aren't going to maintain it like the previous owner.

**Comment by mherrmann** [comment]
Feel free to submit a PR that removes the feature.

**Comment by PotentCricket** [comment]
> Feel free to submit a PR that removes the feature.

What an arrogant response. If you couldn't be bothered to maintain the project, why take over it?!

I won't be using it again, can't imagine many others will either with the attitude you have towards users.

**Comment by mherrmann** [comment]
I am providing you with a free service. It is not perfect. But still, there are some features that are usable. 

You feel entitled to a fix, and insult me as arrogant. 

How can you hold this position? What gives you the right to demand anything from me?

**Comment by PotentCricket** [comment]
> I am providing you with a free service. It is not perfect. But still, there are some features that are usable. 
> 
> 
> 
> You feel entitled to a fix, and insult me as arrogant. 
> 
> 
> 
> How can you hold this position? What gives you the right to demand anything from me?

1, nobody is demanding anything from you. Users have simply requested a fix on a feature that worked perfectly fine before you took over, it's just a general expectation from users that if you took over you'd be properly maintaining the repo I guess, weird right?!

2, as I said, if you had no intention on maintaining why take over in the first place.

3, you didn't provide anyone with anything, the previous owner built this and you've taken over and let things start breaking. 

4, I quite literally don't care anymore, I won't be using it. Good luck with your half maintained repo 👍 🤷‍♂️

**Comment by mherrmann** [comment]
Yes, goodbye. Happy not to see you here again :wave:

**Comment by KutaG59** [comment]
> You feel entitled to a fix, and insult me as arrogant.


1. No one feels entitled but you, and its obvious. We expect someone to maintain the exact functionality when being transferred to a new owner, unless said owner has plans to change how the application/website works. No where, on the site, nor this repo, do I see a SINGLE comment from you, about a roadmap, or plans to do such changes. Stop being an asshole, you don't look good, and no one will enjoy your "free service" with the entitlement you act of.
2. You are arrogant. You are the worst Dev ive personally met yet, I wouldn't want to work with you, or near you ever. I imagine your frail ego makes you feel the same just because of my own opinion on you.  

> 
> arrogant
> 
> /ăr′ə-gənt/
> 
>     Making or having the disposition to make unwarrantable claims of rank or estimation; giving one's self an undue degree of importance; aggressively haughty; full of assumption: applied to persons.
>     Characterized by arrogance; proceeding from an overestimate of one's importance or superiority to others: applied to things: as, arrogant claims.
>     Synonyms Authoritative, Magisterial, Dogmatic, etc. (see magisterial), proud, assuming, overbearing, presumptuous, supercilious, lordly, cavalier, important, swelling, blustering, grand, disdainful, overweening. 

As you will be happy to not see me here, I will be happy to know your ownership will be nothing but ineffectual. You did nothing good for this website, and its sad that you continue to behave the way you do. Good luck with your future projects, assuming anyone wants to work with such a Dev.

**Comment by mherrmann** [comment]
I didn't read your whole message @KutaG59, but I gleaned that you are leaving. That's good. Goodbye 👋

**Comment by st-jliu** [comment]
Dear viveksh1:

This project was taken over by Splashtop now, we will continue the development and maintenance, we re-design the whole packs feature, you are welcome to provide further feedback and suggestions.
